File: src/main/java/io/github/oleksivio/telegram/bot/api/model/method/message/EditOtherMessageMedia.java
package io.github.oleksivio.telegram.bot.api.model.method.message;

import io.github.oleksivio.telegram.bot.core.controller.network.ActionNetworker;
import io.github.oleksivio.telegram.bot.core.model.method.message.EditMessageMedia;
import io.github.oleksivio.telegram.bot.core.model.responses.BooleanResponse;
import io.github.oleksivio.telegram.bot.core.model.responses.CommonResponse;

public class EditOtherMessageMedia extends EditMessageMedia<Boolean> {

    public EditOtherMessageMedia(ActionNetworker actionNetworker) {
        super(actionNetworker);
    }

    @Override
    protected Class<? extends CommonResponse<Boolean>> getResultWrapperClass() {
        return BooleanResponse.class;
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/method/message/EditOtherMessageMedia.kt
package io.github.oleksivio.telegram.bot.api.model.method.message

import io.github.oleksivio.telegram.bot.core.controller.network.ActionNetworker
import io.github.oleksivio.telegram.bot.core.model.BooleanResponse
import io.github.oleksivio.telegram.bot.core.model.method.message.EditMessageMedia

class EditOtherMessageMedia(actionNetworker: ActionNetworker) : EditMessageMedia<Boolean>(actionNetworker) {

    override val resultWrapperClass = BooleanResponse::class
}
